site/_includes/layouts/blog-post.njk

```diff
@@ -28,6 +28,10 @@
   <div class="display-flex gap-top-300 lg:gap-top-400">
     <div class="display-flex justify-content-center width-full">
       <article class="post stack measure-long width-full pad-left-400 pad-right-400">
+        {% if newMaterialURL %}
```

Comment:
All of our multi-word YAML keys are snake case (by it being related to Python): `stack_overflow_tag`, `project_icon_color`, all of i18nk keys,
`project_key`, `origin_trial`. So it would be good to follow that scheme for this one. `new_material_url` also sounds quite specific. `new_material` would already work for me.

Reply:
I have updated YAML frontmatter key from camelcase to underscored